src/app/retrospective/retrospective-feedback/retrospective-feedback.component.ts

```diff
@@ -301,19 +316,23 @@ export class RetrospectiveFeedbackComponent implements OnInit, OnChanges, OnDest
         }
     }
 
-    getTeamMemberOptions(teamMembers) {
+    getTeamMemberOptions() {
+        let teamMembers = this.teamMembers;
         teamMembers = _.map(teamMembers, (data: any) => {
             return {
                 id: _.parseInt(data.ID),
                 value: (data.FirstName + ' ' + data.LastName).trim(),
             };
         });
-        return [{ id: null, value: 'None' }, ...teamMembers];
+        return teamMembers;
     }
 
-    revertCellValue(params) {
+    revertCellValue(params, assigneeIDPrevValue) {
```

Comment:
We should use rest parameters here, and handle based on conditions.
Refer this for more details, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters

Reply:
done